### DISCUSSION TITLE
Need Help Installing Kavita

### DISCUSSION THREAD

**Discussion by maximo1984** [Kareadita/Kavita#2833] [Ideas]
### Idea Description

I had issues prior with Kavita not scanning in new titles when I did a "scan library" or "force scan". So I decided to try and install fresh from docker. I followed all the steps but was unable to figure out what the path was to an internal drive where my files are when writing the "nano docker-compose.yml" file. So I opted to use the other route by downloading the tar.gz file for my distro (Ubuntu), unzipping it and starting fresh as I did originally. however, when I go to the 5000 port, I get this message.
![Screenshot from 2024-04-07 21-40-59](https://github.com/Kareadita/Kavita/assets/40921347/1d017eec-799a-4a0c-9932-cdd90f458322)

Any help would be appreciated. I've been a $10 a month contributor for a couple months now, but have been unable to use it. If I cannot get it going, I'll have to just part ways.

Thanks

### Idea Category

User Experience

### Duration of Using Kavita

3 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by maximo1984** [comment]
In my opinion, I believe it may still be loading from the docker but I am not sure. I attempted to uninstall docker using "docker stop kavita" and "docker rm kavita" but I just get errors. "permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Delete "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/kavitas": dial unix /var/run/docker.sock: connect: permission denied
"

**Reply by therobbiedavis** [reply]
That is indeed loading the docker image from the old registry.

Please try `sudo docker stop kavita`

Or just update the image to `jvmilazz0/kavita:latest`

**Reply by maximo1984** [reply]
How can I update the image? I followed the link but it wasn't very clear what exactly to do.

**Reply by therobbiedavis** [reply]
It's not clear how you initially set up kavita via docker. Did you use `docker run...` or did you use docker compose? If docker run you'd need to remove the old container and rerun the docker run command editing the volumes as appropriate.

https://wiki2.kavitareader.com/installation/docker

**Comment by maximo1984** [comment]
It's all good, sudo prefix worked. Started traditional way. Library scanning in now.
